run_brer/run_config.py

```diff
@@ -377,18 +377,25 @@ def __train(self, tpr_file=None, **kwargs):
             # of alpha.
             # noinspection PyUnresolvedReferences
             current_alpha = context.potentials[i].alpha
+            if current_alpha == 0.0:
+                raise RuntimeError('Alpha value was constrained to 0.0, which indicates something went wrong')
+
             # noinspection PyUnresolvedReferences
             current_target = context.potentials[i].target
+            current_converged = getattr(context.potentials[i], 'converged', None)
 
             self.run_data.set(name=current_name, alpha=current_alpha)
             self.run_data.set(name=current_name, target=current_target)
+            self.run_data.set(name=current_converged, converged=current_converged)
             self._logger.info("Plugin {}: alpha = {}, target = {}".format(current_name,
```

Comment:
At a minimum, you need to add the field that will get formatted with this extra value.

Optionally, you could do something more fancy, like constructing the message string to only mention "converged" if it is not `None`, but that's up to you. :-)

Reply:
Yeah, I was thinking about this earlier and tbh, if `converged` is more under-the-hood, I think it's not necessary to log. The assumption can be that unless you receive an error stating otherwise, `converged` is not a notable parameter to track?